README.md

```diff
@@ -92,8 +92,8 @@ To install the project from source (recommended for developers only).
 ```bash
 git clone https://github.com/siliconcompiler/siliconcompiler
 cd siliconcompiler
-pip install -r requirements.txt
 python -m pip install -e .
+python -m pip install -e .[docs,test]  # Install dependencies for generating docs and running tests
```

Comment:
Could technically skip the bare `python -m pip install -e .`, unless the idea is to give someone a path forward if the docs/test deps install fail?

Reply:
The idea was to give regular users the basic choice and developers the added sections. Maybe that is not 100% clear

Reply:
I think it _would_ be more clear to update to this:
```python -m pip install -e .              # Required install step```
```python -m pip install -e .[docs,test]  # Optional install step for generating docs and running tests```